tests/helper/broker_test.cpp

```diff
@@ -249,6 +249,34 @@ TEST(BrokerTest, RecvRequestInit)
     EXPECT_STREQ(std::string_view(pv[1]).data(), "arachni.com");
 }
 
+TEST(BrokerTest, RecvRequestInitOverLimits)
+{
+    mock::socket *socket = new mock::socket();
+    network::broker broker{std::unique_ptr<mock::socket>(socket)};
+
+    std::stringstream ss;
+    msgpack::packer<std::stringstream> packer(ss);
+    packer.pack_array(2);
+    pack_str(packer, "request_init");
+    packer.pack_array(1);
+    packer.pack_map(257);
+    for (unsigned i = 0; i < 257; i++) {
```

Comment:
I'd suggest to move the `257` to a var with a name to it shows what it represents as it is a very random number for someone who did not know the problem